docroot/themes/humsci/humsci_basic/package.json

```diff
@@ -14,6 +14,10 @@
     "last 2 versions",
     "ie 11"
   ],
+  "dependencies": {
+    "bourbon": "^5.1.0",
+    "decanter": "^5.1.0"
```

Comment:
@sherakama is decanter 5.1 a good latest version to bring in? Nathan saw that the latest is an alpha release

Reply:
Copying the slack response here:

> As with any major version difference there will be work on your end to adjust from 5 to 6. We are planning to release the first stable version of 6 tomorrow. Nothing too major in version 6 I hope but I haven't put together the full change docs yet. If you are already in flight on your work you are safe to stay on version 5. We will release a final version with all the deprecations for that as well. When you are ready you can then look at the effort to update and plan accordingly.

Reply:
Thanks Shea!